src/pages/documentation/adobe-campaign-standard/migration.md

```diff
@@ -1,3 +1,12 @@
+---
+title: Migrating to AEPCampaign reference
+description: Learn how to migrate to the AEPCampaign APIs from the ACPCampaign APIs.
+keywords:
+- Adobe Campaign
```

Comment:
Campaign 
Campaign Standard

Reply:
Same comment here - since the full name _is_ Adobe Campaign + Adobe Campaign Standard, I'm using the full names here.